application/views/admin/super/fullpagebar_view.php

```diff
@@ -99,6 +99,29 @@ class='btn btn-default'
                 </a>
             <?php endif; ?>
 
+            <!-- Reset Survey Menus -->
+            <?php if (isset($fullpagebar['menus']['buttons']['resetMenus']) && $fullpagebar['menus']['buttons']['resetMenus']) :?>
```

Comment:
I always prefer : `if (!empty($fullpagebar['menus']['buttons']['resetMenus'])`

shorter, and allow 0/""/null/false/not set for false

Like other dev want :)